config/job_conf.yml.k8s_interactivetools

```diff
@@ -0,0 +1,41 @@
+execution:
```

Comment:
This and `galaxy.yml.k8s_interactivetools` would be interesting as an example with some prose around it in a new document (`doc/source/admin/interactivetool_setup.md` ?), but so far our procedure is to add new options in `job_conf.xml.sample_advanced`, and I don't think we should stop doing that now. I get this is easier to express in yml, but hopefully we'll move the entire `job_conf.xml.sample_advanced` document over to `job_conf.yml.sample_advanced`. Having 2 documents to look into would make that task more difficult for when we're going to do that and I think it is also better for admins to have a complete listing of options in one place.

Reply:
There was already a document for the non-k8s ITs, I just added these on the side to make it easier and clearer that you can run ITs without the reverse proxy, and without having to dig in the giant advanced files, which I assumed what the original `.interactivetools` files were for. I personally don't care though it was just suggested that having example configurations might be helpful for some, so I'm fine just removing them altogether, although I would say maybe we should remove the other ones too then, as to not imply that there is only one way to run ITs?

Reply:
There was already a conf file for the non-k8s ITs, I just added these on the side to make it easier and clearer that you can run ITs without the reverse proxy, and without having to dig in the giant advanced files, which I assumed what the original .interactivetools files were for. I personally don't care though it was just suggested that having example configurations might be helpful for some, so I'm fine just removing them altogether, although I would say maybe we should remove the other ones too then, as to not imply that there is only one way to run ITs?

Reply:
Yeah, I think the other ones should be removed as well, I can open another PR to do that. `job_conf.xml.sample_advanced` is an almost complete example of how you can configure almost anything related to Galaxy's job config. I agree that's not great if you're trying to find out how to do something specific, that's why I suggested writing `doc/source/admin/interactivetool_setup.md` with more details on the settings and what they do.